### ISSUE TITLE
Potential license conflict derived from the MMDVMHost GPL contradiction prohibiting commercial use.

### ISSUE THREAD

**Issue by scalahater** [nostar/dudestar#43]
TL;DR: MMDVMHost is not GPLv2+.

I observed that dudestar uses several files from https://github.com/g4klx/MMDVMHost project, however, when one reads the license of that project https://github.com/g4klx/MMDVMHost/blob/master/README.md, it is stated that MMDVMHOST:

> This software is licenced under the GPL v2 and is intended for amateur and educational use only. Use of this software for commercial purposes is strictly forbidden.

Some of the files present in dudestar are authored by "Jonathan Naylor G4KLX".

Some people have opened tickets and raised their concern to https://github.com/g4klx/MMDVMHost/issues/373 but they have been ignored and the license conflict still persists.

It cannot be stated that something is covered by the GPL and at the same time forbidden for commercial purposes.

![Alt Text](https://media.giphy.com/media/EldfH1VJdbrwY/giphy.gif)

By incorporating part of the MMDVMHost, this conflict/ambiguity/contradiction affects dudestar.

**Comment by nostar** [comment]
oh well, i couldn't care less.

**Comment by scalahater** [comment]
![dudestar](https://user-images.githubusercontent.com/14271486/100803707-e67b2400-342b-11eb-8d77-9c7a76545f54.gif)
